tests/contrib/sqlalchemy/models.py

```diff
@@ -15,7 +15,7 @@ class Author(AuditBase):
     """The Author domain object."""
 
     name: Mapped[str]
-    dob: Mapped[date | None]
+    dob: Mapped[date] = mapped_column(nullable=True)
```

Comment:
What's the issue with this one? The annotated type no longer matches the column type, right?

Reply:
That's correct.  I couldn't get it to accept `Mapped[date|None]` in all Python version.  It also complained about `Mapped[Optional[date]]`, so I opted to use the override in `mapped_column`